### PR TITLE
change "get minions" to "get nodes"  in docs

### PR DESCRIPTION
**Pull request by jiangyaoguo**
Now the "get minions" has been changed to "get nodes". There're still some where unchanged in docs.

### CODE REVIEW

**Comment by nikhiljindal** [comment]
@k8s-bot ok to test

**Comment by nikhiljindal** [comment]
This change LGTM, but while we are at it, I will recommend updating all references to minions in that doc to say nodes.

**Comment by nikhiljindal** [comment]
cc @bgrant0607

**Comment by jiangyaoguo** [comment]
@nikhiljindal @JanetKuo Since JanetKuo has done the works in vagrant.md, I 'll change the other docs.

**Comment by jiangyaoguo** [comment]
@nikhiljindal

**Comment by nikhiljindal** [comment]
Sorry needs rebase

File: docs/getting-started-guides/fedora/fedora_ansible_config.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ The ansible scripts are quite hacky configuring the network, you can see the [RE
 
 **Configure the ip addresses which should be used to run pods on each machine**
 
-The IP address pool used to assign addresses to pods for each minion is the `kube_ip_addr`= option.  Choose a /24 to use for each minion and add that to you inventory file.
+The IP address pool used to assign addresses to pods for each node is the `kube_ip_addr`= option.  Choose a /24 to use for each node and add that to you inventory file.
```
Comment (nikhiljindal) [inline docs/getting-started-guides/fedora/fedora_ansible_config.md]: add that to *you* inventory -> add that to *your* inventory

**Comment by nikhiljindal** [comment]
changes LGTM

**Comment by jiangyaoguo** [comment]
@nikhiljindal Thanks a lot for your review. I have rebased it and made a change.

**Comment by nikhiljindal** [comment]
Shippable had timed out. Restarted it

**Comment by jiangyaoguo** [comment]
@nikhiljindal  Do I can restart shippable check manually?

**Comment by jiangyaoguo** [comment]
@nikhiljindal  How can I restart shippable check manually?

**Comment by nikhiljindal** [comment]
I did it.

For next time, you can click on "Details" next to shippable, which will take you the build for this PR on app.shippable.com.
There you will see a refresh like button, which will say "Re-run Build" when you hover over it. The button is towards top-right on that page.

**Comment by bgrant0607** [comment]
Thanks much for the change.  @thockin please resolve overlap with #11130

**Pull request merged**